<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QV_Users</class>
 <widget class="QWidget" name="QV_Users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benutzereinstellungen</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Übersicht</string>
    </attribute>
    <widget class="QTableWidget" name="tbl_users">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>951</width>
       <height>421</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Passwort</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Administrator</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_details">
    <attribute name="title">
     <string>Details</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Passwort:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_root">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Administrator</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_name">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="tb_password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_abort">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zurücksetzen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Übernehmen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hinzufügen</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
